helpdesk/helpdesk/settings.py

```diff
@@ -39,6 +40,12 @@
         ) from None
     raise
 
+sentry_sdk.init(
+    dsn=getattr(configuration, "SENTRY_DSN", None),
+    traces_sample_rate=getattr(configuration, "SENTRY_TRACES_SAMPLE_RATE", 1.0),
+    profiles_sample_rate=getattr(configuration, "SENTRY_PROFILES_SAMPLE_RATE", 1.0),
```

Comment:
Tracing and profiling jumps the usage quota a fair bit, and for this project isn't really something we care about. Probably best leaving these unset

Reply:
We can just set it to 0

Reply:
b219f7d